src/pages/api/loadQuestionList.ts

```diff
@@ -5,16 +5,35 @@ import { ID } from 'src/types/common'
 export interface LoadQuestionListParams {
   cid: ID
   topic?: string
+  page: number
```

Comment:
doesn't page keep changing whenever new questions are added?

Reply:
> doesn't page keep changing whenever new questions are added?

the questions in each page change position based on when they were last updated if they were updated recently they appear earlier than the ones that are updated long time ago https://github.com/kixlab/kuiz/pull/79/commits/841bcfce27c6eed67644457347095f96c95eb0a5